src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Languages/en.xaml

```diff
@@ -9,6 +9,6 @@
     <system:String x:Key="Microsoft_plugin_indexer_name">Name</system:String>
     <system:String x:Key="Microsoft_plugin_indexer_path">Path</system:String>
     <system:String x:Key="Microsoft_plugin_indexer_drivedetectionwarning">Warning: Detected that not all drives are indexed.</system:String>
```

Comment:
Super super minor nit: do we need 'Detected'?

A.k.a. "Warning: not all drives are indexed".

Reply:
Fair, simplier

Reply:
Will modify that. Thanks!

Reply:
Done.
![image](https://user-images.githubusercontent.com/28739210/88209249-486a8800-cc07-11ea-8383-f963cc53b675.png)